packages/create-bison-app/template/package.json.ejs

```diff
@@ -46,25 +46,25 @@
     "watch:ts": "yarn dev:typecheck --watch"
   },
   "dependencies": {
-    "@chakra-ui/pro-theme": "^0.0.57",
-    "@chakra-ui/react": "^2.2.0",
-    "@chakra-ui/theme": "^2.1.0",
-    "@emotion/react": "^11.0.0",
-    "@emotion/styled": "^11.0.0",
     "@next-auth/prisma-adapter": "^1.0.5",
-    "@nikolovlazar/chakra-ui-prose": "^1.2.1",
     "@prisma/client": "^4.8.0",
+    "@radix-ui/react-label": "^2.0.1",
+    "@radix-ui/react-navigation-menu": "^1.1.2",
+    "@radix-ui/react-toast": "^1.1.3",
     "@tanstack/react-query": "^4.16.1",
     "@tanstack/react-query-devtools": "^4.16.1",
     "@trpc/client": "^10.0.0",
     "@trpc/next": "^10.0.0",
     "@trpc/react-query": "^10.0.0",
     "@trpc/server": "^10.0.0",
     "bcryptjs": "^2.4.3",
+    "class-variance-authority": "^0.5.1",
+    "clsx": "^1.2.1",
     "cross-fetch": "3.0.5",
     "framer-motion": "^8.5.0",
```

Comment:
`framer-motion` was also only installed because it was required for chakra. I think we can remove.